src/morphology_workflows/tasks/repair.py

```diff
@@ -235,16 +235,22 @@ def inputs(self):
         }
 
 
-class SmoothDiameters(StrIndexMixin, SkippableMixin(True), ElementValidationTask):
+class SmoothDiameters(StrIndexMixin, SkippableMixin(True), SetValidationTask):
     """Smooth diameters with :mod:`diameter_synthesis`.
 
     We use actual diameters to learn a diameter model used to diametrize the morphology.
 
     By default, this task is skipped.
     """
 
+    neurite_types = luigi.OptionalListParameter(
+        default=None, description=":list: List of neurite_types to smooth"
```

Comment:
I don't remember if it's valid but it would be nice to set a more complete type in the description like `:list[str]:`.
Also, you could add a list of possible values using the `schema` parameter.

Reply:
bah, it is standard neurite_type thing we see everywhere, if a user needs this, he/she will know what this is

Reply:
Ok, we can see later for that